src/parser_urdf_TEST.cc

```diff
@@ -583,6 +598,55 @@ TEST(URDFParser, CheckFixedJointOptions_preserveFixedJoint)
   ASSERT_EQ(jointType, "fixed");
 }
 
+/////////////////////////////////////////////////
+TEST(URDFParser, CheckParserConfig_preserveFixedJoint)
+{
+  // Convert a fixed joint with only preserveFixedJoint
+  // (i.e. converted to fixed joint)
+  std::ostringstream fixedJointPreserveFixedJoint;
+  fixedJointPreserveFixedJoint << "<robot name='test_robot'>"
+    << "  <link name='link1'>"
+    << "    <inertial>"
+    << "      <origin xyz='0.0 0.0 0.0' rpy='0.0 0.0 0.0'/>"
+    << "      <mass value='1.0'/>"
+    << "      <inertia ixx='1.0' ixy='0.0' ixz='0.0'"
+    << "               iyy='1.0' iyz='0.0' izz='1.0'/>"
+    << "    </inertial>"
+    << "  </link>"
+    << "  <link name='link2'>"
+    << "    <inertial>"
+    << "      <origin xyz='0.0 0.0 0.0' rpy='0.0 0.0 0.0'/>"
+    << "      <mass value='1.0'/>"
+    << "      <inertia ixx='1.0' ixy='0.0' ixz='0.0'"
+    << "               iyy='1.0' iyz='0.0' izz='1.0'/>"
+    << "    </inertial>"
+    << "  </link>"
+    << "  <joint name='joint1_2' type='fixed'>"
+    << "    <parent link='link1' />"
+    << "    <child  link='link2' />"
+    << "    <origin xyz='0.0 0.0 0.0' rpy='0.0 0.0 0.0' />"
+    << "  </joint>"
+    << "</robot>";
+
+  // Check that there is a fixed joint in the converted SDF
+  sdf::SDF fixedJointPreserveFixedJointSDF;
+
+  // Set the config option to preserve fixed joints.
+  sdf::ParserConfig config_;
```

Comment:
style nit: I don't think we need the `_` suffix.

Reply:
Done